crates/cubecl-spirv/src/instruction.rs

```diff
@@ -323,6 +323,9 @@ impl<T: SpirvTarget> SpirvCompiler<T> {
                 }
             }
             Operator::Select(op) => self.compile_select(op.cond, op.then, op.or_else, out, uniform),
+            Operator::ConditionalRead(op) => {
+                self.compile_select(op.cond, op.slice, op.fallback, out, uniform)
```

Comment:
Is it correct? I think this is more than a select, since the select doesn't have a read position? I guess it should do the same thing as the index operation.